lib/tasks/serve.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Mix.Tasks.Serve do
+  use Mix.Task
+
+  def run([port_str]) do
```

Comment:
Derp, didn't see the bin.

So the procfile has the port pushed in (using `$PORT`). I assume we need to push this through to the server? (It's not an ENV...)